Milestone1: first page layout

BertrandConxy opened the pull request:
#  Created first-page layout
in this repository:
- I built a mobile version page layout
- the first 2 sections.
- all using flexbox
- background image has been copied from the Figma design plus social media icons.

BertrandConxy committed:
Create html and css files

BertrandConxy committed:
Update html and css

Made a  mobile version layout of the project

BertrandConxy committed:
Update html and css

BertrandConxy committed:
Create linters.yml

t-yanick commented on index.html:
```diff
@@ -0,0 +1,33 @@
+    </article>
+
+</main> 
+</div>
```
- [ ] As per [HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), your indentation is not correctly done. Check the `<div class="wrapper">`, `<header class="header">` tags to make sure they are well indented with respect`<body>` element. 
![image](https://user-images.githubusercontent.com/62849872/143443912-d474e9f7-0a57-4178-9842-84e954ea347f.png)
![image](https://user-images.githubusercontent.com/62849872/143443959-0a0b2744-599a-460d-9af3-d73b182714fa.png)

t-yanick commented on style.css:
```diff
@@ -0,0 +1,106 @@
+.wrapper {
```
- [ ] You have done so well implementing the design as per the figma template, but looking at the `right margin`, there's quite a remarkable difference from the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988). Please also try to match the colors and font sizes. 
- Your design 
![image](https://user-images.githubusercontent.com/62849872/143445170-a1ec7cce-8e85-4c88-a441-1924850961e7.png)

- Figma design 
![image](https://user-images.githubusercontent.com/62849872/143445284-244646ef-fec8-440b-84dd-f4e6689bea3e.png)

I think you need to review the properties applied to the `wrapper` class. Comment out the wrapper class and see the display of your page in the browser and you will have an idea how to fix it.

BertrandConxy replied:
Hello @t-yanick. Am happy with your reviews and I promise you that I am going to make changes that you have told me to do as soon as possible. Thank you.

t-yanick reviewed (changes requested):
Hi @BertrandConxy , 

Good job so far!

**Good Points** 👍 
- Project is setup correctly.
- Descriptive Pull Request
- No linter errors
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

Your README file is not loaded for inline comments, as such I will provide feedback here in the summary section.

- [ ] You have followed the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) for your README file, but you missed some very important elements. Your README file does not indicate how to use and run your project. This is very necessary for users visiting your repo and will want to use and run your project. Kindly update your README file to have these items. Also consider adding a screenshot of your project and if possible add a live demo link.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Always use descriptive names for your branches as this makes your repo professional (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars)
- Capitalising your Pull request title (e.g. Milestone1: Portfolio Page Layout)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BertrandConxy committed:
Update README.md

BertrandConxy committed:
Update README.md

BertrandConxy committed:
Fix right margin, colors and indentation

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update README.md

BertrandConxy commented:
## Made Changes on the First Page Layout
After reviewing all the comments, I have made the following changes in my Repo:
-Updated my README.md file where I added how to use and run the project, added a screenshot of the project, and a live demo link.
-I Updated the index.html file where I made the best Indentation of the body tag and header tag and also the other tags, I removed the wrapper div so that I can solve the problem of unproper right margin.
- Also I replaced the article tag with a section tag.
- I made some changes in the style.css file to
- Now it looks like this:
![Milestone1](https://user-images.githubusercontent.com/90222110/143534511-4363c1c7-a13b-456b-9acb-c9998b76c7a3.jpg)


Please, this is my second review request, I humbly request that you may reply to me swiftly. Thank you.

Cathella commented on index.html:
```diff
@@ -8,26 +8,25 @@
+                <div class="logo"><h1>Welcome</h1></div>
+                <img class="ToggleButton" src="/The-Portfolio/Milestone1/images/icons/icon-Menu.png" alt="togglebutton">
+            </nav>
+        </header>
```
**_Since this requirement wasn't mentioned by the first reviewer, it is marked as [optional] but it is highly recommended that you implement it_**

- [ ] [Optional] You have done a great job structuring the header section 👏🏻. However, we can structure it even better as per project requirements. The recommended structure is as below

```
<header>
  <a href="#"> -- logo goes here -- </a>
  <nav>
    <button> -- menu icon goes here -- </button>
  </nav>
</header>
```

Cathella commented on index.html:
```diff
@@ -8,26 +8,25 @@
+                 don’t hestiate to contact me.</p></div>
         
-        <img class="social-logos" src="images/icons/socialMediaIcons.png" alt="Social media icons">
+            <img class="social-logos" src="/The-Portfolio/Milestone1/images/icons/socialMediaIcons.png" alt="Social media icons">
```
- [ ] Adding `/The-Portfolio/Milestone1/` to the `src` of the image made it not possible to be loaded as seen below. I therefore suggest leaving it this way `src="images/icons/socialMediaIcons.png"`.

<img width="342" alt="wel" src="https://user-images.githubusercontent.com/4247740/143552282-20143610-dc91-4e2d-bcba-0fd8ed3b329f.PNG">

**_Since this requirement wasn't mentioned by the first reviewer, it is marked as [optional] but it is highly recommended that you implement it_**

- [ ] [Optional] The way you added the social media icons was not right. This project requires you to have - The list of icons that uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (a tag)

Cathella reviewed (changes requested):
## CHANGES REQUIRED 🟡 

### Hi @BertrandConxy 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**_Since this requirement wasn't mentioned by the first reviewer, it is marked as [optional] but it is highly recommended that you implement it_**

- [ ] [Optional] The figma design has an extra feature that has not been implemented in yours as seen below. I there fore suggest that you go on and add that down arrow so that your design matches with the original figma design.

<img width="286" alt="ffg" src="https://user-images.githubusercontent.com/4247740/143555856-a7a3d97d-293e-4b60-9e13-09ea764229a0.PNG">


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BertrandConxy commented:
Thanks @Cathella. I am going to make changes as requested.

BertrandConxy committed:
Update index.html and style.css

BertrandConxy committed:
Modified style.css

BertrandConxy committed:
Fix linter errors

BertrandConxy commented:
## Made changes for first-page layout
I have made the following changes:
- Added the arrow button just below social media icons
- corrected the structure of the header and nav tag.
![header nav](https://user-images.githubusercontent.com/90222110/143579145-ca7fc002-3af7-45df-8141-d9b8a9548aca.jpg)
-Also I corrected the img src path. You can see it in the screenshot above.
- I wrapped the social media icons in the list tag.
-
![list](https://user-images.githubusercontent.com/90222110/143579403-71d28613-3c67-4a12-898d-4d50eb782547.jpg)
 
-now the final looks like this:
![headline1](https://user-images.githubusercontent.com/90222110/143579442-e1ee7210-03db-46db-aaa7-1507da8cf240.jpg)
s
- Please I humbly request that you may reply swiftly so that I may continue with the other milestone. Thansks

enaburekhan commented on index.html:
```diff
@@ -0,0 +1,44 @@
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+      <div class="nav">
+        <header>
+            <div><a href="#" class="logo"><h1>Welcome</h1></a></div>
+        </header>
+        <nav>
+                
+            <div class="Togglebutton"><a href="#"><img class="MenuIcons" src="images/icons/icon-Menu.png" alt="togglebutton"></a></div>
+        </nav>
+      </div> 
```
**Optional**

- [ ] As suggested by the last reviewer. It will be nice if we follow the project requirement. This enables us to use Semantic HTML for best practices. 

```
<header>
  <a href="#"> -- logo goes here -- </a>
  <nav>
    <button> -- menu icon goes here -- </button>
  </nav>
</header>
```
The `header tag` should wrap the content, not the current `div tag`

enaburekhan commented on index.html:
```diff
@@ -0,0 +1,44 @@
+        </header>
+        <nav>
+                
+            <div class="Togglebutton"><a href="#"><img class="MenuIcons" src="images/icons/icon-Menu.png" alt="togglebutton"></a></div>
```
- [ ] As also suggested by the last reviewer, the `Togglebutton` is yet to load properly on the browser.

![image](https://user-images.githubusercontent.com/51296741/143586349-3ae4f0bf-9a3d-4f49-8e86-30010cefaf0a.png)

Change the current path to `src="images/Icons/Icon-Menu.png"` so it can properly grab the image.

enaburekhan commented on index.html:
```diff
@@ -0,0 +1,44 @@
+        
+                 <ul class=social>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/twitter.png" alt="Twitter icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/Linkedinicon.png" alt="Linkedin icon"></a></div></li>
```
**Optional**

- [ ] I will suggest you add the `Linkedinicon.png` icon to the images folder. So the browser can grab and load it. This is what we have currently:

![image](https://user-images.githubusercontent.com/51296741/143587250-5bc1341b-be99-4ba1-8204-b92dc7f940af.png)

enaburekhan reviewed (changes requested):
Hi BertrandConxy, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

BertrandConxy committed:
Changes asked

BertrandConxy committed:
Add files via upload

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update index.html

BertrandConxy committed:
upadate

BertrandConxy committed:
update

BertrandConxy committed:
Merge branch 'Milestone1' of https://github.com/BertrandConxy/The-Portfolio into Milestone1

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update index.html

BertrandConxy committed:
Update index.html

BertrandConxy commented:
## Made last changes as required
- I fixed the toggle-button image
- fixed the Twitter icon
- used the best HTML and CSS practices

BertrandConxy committed:
Update style.css

BertrandConxy committed:
UPdate

BertrandConxy committed:
Correct Html proper standards

BertrandConxy committed:
Update README.md

VanessaAoki reviewed (commented):
Hi @BertrandConxy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have asked for a review in a Pull Request that was already closed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding! 👩‍💻 

![keep trying](https://media4.giphy.com/media/JpkKuYQ80N9fAXYaxf/giphy.gif)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
